### PR TITLE
Mobile - Authentication screens

### PR DESCRIPTION
**Pull request by emhat098**
- Convert the authentication wireframe to a Flutter UI.

Note: This is UI only; the global state setup or backend integration for completing the full authentication workflow in the mobile app will be done later.

### CODE REVIEW

**Review by panoti — CHANGES_REQUESTED** [review]
avoid push dist/build/cache files to git. You can use gitignore to skip commit these files

**Review by tuonglevan — DISMISSED** [review]
LGTM

File: .DS_Store
Comment (panoti) [inline .DS_Store:1]: Can we ignore .DS_Store due to MacOS?

Comment (emhat098) [reply .DS_Store:1]: I updated the code change. Could you please review the code change?

**Review by panoti — APPROVED** [review]
LGTM

**Pull request merged**